app/views/gettingStarted.scala.html

```diff
@@ -1,40 +1,109 @@
+                    <h3><span>&raquo;</span> Follow the Hello World Tutorial</h3>
+                    <div class="try-option-content">
+                        <p>Play requires Java 8 or higher. You can check your version with the <code>java -version</code> command or install it from <a href="//www.oracle.com/technetwork/java/javase/downloads/index.html" rel="noopener">Oracle's site</a>. See details about <a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">Play's system requirement</a>.</p>
+                        <p>First, download the starter project:</p>
+                            <table>
+                            @examples.sections.map { case (version, section) =>
+                            <tbody>
+                                <tr><th><h4>Play @version Starter Projects</h4></th></tr>
+                                @section.starters.getOrElse("java", Seq.empty).map { (project) =>
+                                    @renderProject(project)
+                                }
+                                @section.starters.getOrElse("scala", Seq.empty).map { (project) =>
+                                    @renderProject(project)
+                                }
+                            </tbody>
+                            }
+                        </table>
+                        <p>Then, unzip the project in a convenient location and follow the directions in the <code>README.md</code> file to build and run the tutorial.</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="try-section">
+                <h2>Already know a bit about Play?</h2>
+                <p>Check out our <a href="//developer.lightbend.com/start/?group=play">example projects for Java or Scala</a>. The downloadable zip files include everything you need, including <a href="//www.scala-sbt.org/index.html">sbt</a>, Play Framework, and an HTTP server. Examples also support <a href="//docs.gradle.org/current/userguide/play_plugin.html">Gradle</a>. You can also create a Play project from the command line using a template if you have sbt installed already.</p>
+                <p>Play requires Java 8 or higher. You can check your version with the <code>java -version</code> command or install it from <a href="//www.oracle.com/technetwork/java/javase/downloads/index.html" rel="noopener">Oracle's site</a>. See details about the <a href="@controllers.documentation.ReverseRouter.latest(None, "Requirements")">requirements to work with Play</a>.</p>
+
+                <div class="try-option">
+                    <h3><span>&raquo;</span> Try a focused example</h3>
+                    <div class="try-option-content">
+                        <p>Follow these steps to try an example project:</p>
+                        <ol>
+                            <li>Choose an example from <a href="https://developer.lightbend.com/start/?group=play">Lightbend's Tech Hub</a>.</li>
```

Comment:
What's the advantage of moving these examples off the download page and onto another site? I think it makes the experience less streamlined.

Reply:
Increase awareness of Tech Hub. But I agree that it is less streamlined. 

But I think we can have both. We can have a streamlined experience here, at the "try play" page, and have other places linking to Tech Hub (and even this page too).

@rstento, what do you think?

Reply:
Can we go ahead with these changes and see what they look like? I always find it helps when you see it live. Then maybe we can revisit?

Reply:
@rstento 👍